src/index.js

```diff
@@ -2,6 +2,7 @@
 /* eslint-disable no-unused-vars */
 /* eslint-disable no-use-before-define */
```

Comment:
Kindly don't disable the linter checks. I would recommend fixing the linter issues instead of just disabling them.